tests/test_build.nf.test.snap

```diff
@@ -2,46 +2,25 @@
     "test_build": {
         "content": [
             [
-                "bedops",
-                "bedops/Homo_sapiens.GRCh38.102_rrna_intervals.gtf.bed",
-                "gatk4",
-                "gatk4/Homo_sapiens.GRCh38.102.dna.primary_assembly.dict",
-                "gffread",
-                "gffread/Homo_sapiens.GRCh38.102.fasta",
+                "fastqc",
+                "fastqc/test_1_fastqc.html",
+                "fastqc/test_1_fastqc.zip",
+                "fastqc/test_2_fastqc.html",
+                "fastqc/test_2_fastqc.zip",
                 "pipeline_info",
-                "references",
-                "references/ensembl",
-                "references/ensembl/Homo_sapiens.GRCh38.102.chr.gtf",
-                "references/ensembl/Homo_sapiens.GRCh38.102.dna.primary_assembly.fa",
-                "references/ensembl/Homo_sapiens.GRCh38.102.dna.primary_assembly.fa.fai",
-                "references/ensembl/Homo_sapiens.GRCh38.102.genepred",
-                "references/ensembl/Homo_sapiens.GRCh38.102.gtf",
-                "references/ensembl/Homo_sapiens.GRCh38.102.refflat",
-                "references/ensembl/Homo_sapiens.GRCh38.102_rrna_intervals.gtf.interval_list",
-                "references/hgnc",
-                "references/hgnc/HGNC-DB-timestamp.txt",
-                "references/hgnc/hgnc_complete_set.txt",
-                "rrnatranscripts",
-                "rrnatranscripts/Homo_sapiens.GRCh38.102_rrna_intervals.gtf"
```

Comment:
@atrigila Sorry for this but I'm having many issues when running the pipeline's test. I wanted to talk to you and @nschcolnicov about that because I don't know which ones are known to fail or succeed